lib/c/criu.c

```diff
@@ -1657,3 +1657,71 @@ int criu_restore_child(void)
 {
 	return criu_local_restore_child(global_opts);
 }
+
+int criu_local_get_version(criu_opts *opts)
+{
+	int ret = -1;
+	CriuReq req	= CRIU_REQ__INIT;
+	CriuResp *resp	= NULL;
+
+	saved_errno = 0;
```

Comment:
saved_errno = errno ? Why do we need this?

Reply:
recv_resp() sets saved_errno to the errno in the case of a failure during recv(). This way we can forward the errno of the RPC communication to the library user.